src/libs/tmdk/src/lib/entity.ts

```diff
@@ -10,10 +9,19 @@ import {
   ListEntitiesCommandOutput,
 } from "@aws-sdk/client-iottwinmaker";
 import { getDefaultAwsClients as aws } from "./aws-clients";
-import { delay } from "./utils";
 
 type EntityDefinition = Omit<CreateEntityRequest, "workspaceId">;
 
+/**
+ * Helper function to wait for set amount of time
+ * @param ms number of ms to wait
+ */
+async function delay(ms: number) {
+  await new Promise((resolve) => {
+    setTimeout(resolve, ms);
+  });
+}
```

Comment:
Why not just use `await delay(100)`? Also this helper function can go in utils

Reply:
Same as above, these were the only 2 functions used in utils, maybe I should recreate the file but remove the rest of the problematic functions instead

Reply:
Agree with this solution